core/src/services/alluxio/backend.rs

```diff
@@ -229,8 +228,7 @@ impl Accessor for AlluxioBackend {
     }
 
     async fn write(&self, path: &str, args: OpWrite) -> Result<(RpWrite, Self::Writer)> {
-        let w = AlluxioWriter::new(self.core.clone(), args, path.to_string());
```

Comment:
Hi, please activate the `write_can_multi` capability to ensure corresponding tests are enabled.

Reply:
fixed